File: M1LAB_Hammers.java
// CIS 115
// M1LAB
// Name and age conversation
// Plan:
// Ask user Name
// Say it back
// Ask age
// Add 1 to it
// Say next age
// Done

//add imports
import java.util.Scanner; // for input

class Main {
  public static void main(String[] args) {
      Scanner s = new Scanner ( System.in);
      System.out.print("Enter your name: ");
      String name=s.next();
      System.out.println("Hello: " + name);
      System.out.print("Enter your age: ");
      int age.s.nextInt();
      int sum = age + 1;
      System.out.println("Your age next year will be " + sum);
  }
}

File: M1LAB_Hammers.py
#This program asks user for name, age, and displays it
#afterwards with their name, age, and what age they
#will be next year.
#

def main():

    #get user name
    name = input('What is your name?')
    #get user age
    age = int(input('What is your age?'))
    #calculate user age next year
    new_age = age + 1
    #display user name and new age
    print('Your name is ', name, 'and you will be ', new_age,' next year.')

#call main function
main()
